src/libraries/System.Collections/src/System/Collections/Generic/PriorityQueue.cs

```diff
@@ -103,17 +124,39 @@ public PriorityQueue(int initialCapacity, IComparer<TPriority>? comparer)
         }
 
         /// <summary>
-        /// Creates a priority queue populated with the specified elements and priorities.
+        ///  Initializes a new instance of the <see cref="PriorityQueue{TElement, TPriority}"/> class
+        ///  that is populated with the specified elements and priorities.
         /// </summary>
+        /// <param name="items">The pairs of elements and priorities with which to populate the queue.</param>
+        /// <exception cref="ArgumentNullException">
+        ///  The specified <paramref name="items"/> argument was <see langword="null"/>.
+        /// </exception>
+        /// <remarks>
+        ///  Constructs the heap using a heapify operation,
+        ///  which is generally faster than enqueuing individual elements sequentially.
```

Comment:
Needed?

Reply:
I'm assuming most users would be want to know what differentiates this method over just calling `Enqueue();Dequeue()`